src/AlgebraicGeometry/ToricVarieties/ToricLineBundles/cech-cohomology.jl

```diff
@@ -0,0 +1,101 @@
+#The following function is called, when algorithm="chambers" is specified for all_cohomologies
+function _all_cohomologies_via_cech(tl::ToricLineBundle)
+  our_maps = transpose.(matrix.(_toric_cech_complex(tl)))
+  X = toric_variety(tl)
+  alternating_sum = abs(sum(binomial(n_maximal_cones(X), k) * (-1)^k for k in dim(X)+2:n_maximal_cones(X); init = 0))
+  alternating_sum *= Int(ncols(our_maps[end]) // binomial(n_maximal_cones(X), dim(X)+1))
+  @req ncols(our_maps[end]) >= alternating_sum "Inconsistency encountered"
+  rks_maps = push!(rank.(our_maps), alternating_sum)
+  return ZZRingElem.([nrows(our_maps[1]) - rks_maps[1]; [nrows(our_maps[k]) - rks_maps[k] - rks_maps[k-1] for k in 2:dim(X)]; ncols(our_maps[dim(X)]) - rks_maps[dim(X) + 1] - rks_maps[dim(X)]])
+end
+
+function _toric_cech_complex(tl::ToricLineBundle)
+  # Extract essential information
+  X = toric_variety(tl)
+  @req is_complete(X) "Toric variety must be complete!"
+
+  # Compute the hyperplane arrangement
+  a_plane = matrix(ZZ, n_rays(X), 1, coefficients(toric_divisor(tl)))
+  sc = cone_from_inequalities(matrix(ZZ, [[-1; zeros(Int, dim(X))]]))
+  H = Polymake.fan.HyperplaneArrangement( HYPERPLANES = [a_plane matrix(ZZ, rays(X))], SUPPORT=sc.pm_cone)
```

Comment:
This uses polymake internals. If top-level functionality is not available at the moment, can we insert a comment here to explain what is going on? Or maybe open an accompanying issue?